sdk/core/azure-core/src/main/java/com/azure/core/http/policy/ExponentialBackoff.java

```diff
@@ -57,6 +58,26 @@ public ExponentialBackoff() {
         this(DEFAULT_MAX_RETRIES, DEFAULT_BASE_DELAY, DEFAULT_MAX_DELAY);
     }
 
+    /**
+     * Creates an instance of {@link ExponentialBackoff}.
+     *
+     * @param options The {@link ExponentialBackoffOptions}.
+     * @throws NullPointerException if {@code options} is {@code null}.
```

Comment:
nit: needs to copy the other throws from the constructor this is calling into

Reply:
This shouldn't be a case anymore after introducing validation into ExponentialBackoffOptions